core/include/tangram/platform.h

```diff
@@ -13,11 +13,15 @@ using UrlCallback = std::function<void(std::vector<char>&&)>;
 using FontSourceLoader = std::function<std::vector<char>()>;
 
 struct FontSourceHandle {
-    FontSourceHandle(std::string _path) : path(_path) {}
+    FontSourceHandle(std::string _path) : pathOrFontName(_path) {}
+    FontSourceHandle(std::string _pathOrFontName, bool _isFontName) : pathOrFontName(_pathOrFontName), isFontName(_isFontName) {}
```

Comment:
Leaving this as a note for future cleanup on this: Using `Url` class here would be cleaner overhaul, only adding a dependency for this class from platform code.

Reply:
As discussed in slack, makes sense to do this cleanup with `fontConfig` work for linux platforms.